src/components/listBox/tests/ListBox.spec.tsx

```diff
@@ -75,6 +77,13 @@ describe('ListBox', () => {
             expect(onDestroySpy).toHaveBeenCalledTimes(1);
         });
 
+        it('should render items with events on onOptionClick', () => {
+            renderListBox();
+
+            (listBoxComponent.find(ItemBox) as any).node.handleOnOptionClick({target: 'target'});

```

Comment:
Here you click on every boxes? Wouldn't it be better to simulate a click on the 2nd box?

Reply:
Fixed, but when we do .node, it target the node from the list. Modified to target the node 2.